Eigenvalues/InversePowerMethod.m

```diff
@@ -0,0 +1,34 @@
+% [USES] Ad-hoc/SIT
+% [USES] Ad-hoc/SST
+% [USES] LU/Doolittle
+function [lambda, y, step] = InversePowerMethod(A, tol, max_iter, miu)
+    % Check if it's a square matrix, otherwise there won't be any eigenvalues
+    [n m] = size(A);
+    if n ~= m
+        disp ('Not square matrix')
+        return;
+    endif
+
+    y = rand(n, 1);
+    I = eye(n);
+    lambda = inf;
+
+    for step = 1 : max_iter
+        % Solving (A - miu * I) * z = y system using Doolittle LU decomposition 
+        [L U z] = Doolittle(A - miu * I, y);
```

Comment:
ai spatiile aiurea iar

Reply:
Nu inteleg de ce se vad spatiile asa aici, in sublime sunt aliniate perfect.

Reply:
Am incercat si cu editorul online de la github, tot aliniate bine se vad

Reply:
ok, accept asta